screwdriver/api/api.go

```diff
@@ -223,6 +235,47 @@ func (c client) PostCommand(specPath string, commandSpec *util.CommandSpec) (*ut
 	return responseSpec, nil
 }
 
+func (c client) ValidateCommand(yamlString string) (*util.ValidateResponse, error) {
+	var (
+		body        *bytes.Buffer
+		contentType = "application/json"
+		err         error
+	)
+	body, err = writeValidateBody(yamlString)
```

Comment:
You do not need to declare body and err by using `var`.
```
body, err := writeValidateBody(yamlString)
```

Reply:
fixed.